pvlib/solarposition.py

```diff
@@ -453,6 +471,63 @@ def _ephem_setup(latitude, longitude, altitude, pressure, temperature):
     return obs, sun
 
 
+def ephem_next_rise_set(time, latitude, longitude, altitude=0,
```

Comment:
would `next_sun_rise_set_pyephem` be more consistent with your preferred naming scheme?

Reply:
Yes, I'll change.  I have in mind `Location.get_next_sunrise`, `Location.get_previous_sunrise` and wrapper methods.